be/src/runtime/exec_env.h

```diff
@@ -93,6 +93,7 @@ class ExecEnv {
     // declarations for classes in scoped_ptrs.
     ~ExecEnv();
 
+    bool is_init() { return _is_init; }
```

Comment:
please add const to make is_init() read-only

Reply:
OK